### DISCUSSION TITLE
What time support laravel 9

### DISCUSSION THREAD

**Discussion by hinet** [spatie/laravel-permission#2042] [Q&A]
What time support laravel 9

**Comment by erikn69** [comment]
Laravel 9.x supported since the release(laravel-permission v5)
https://github.com/spatie/laravel-permission/blob/20dd894eb64e5b02de9dfa28aa13af0643b2e456/composer.json#L26-L29

**Reply by erikn69** [reply]
@hinet mark this as **answered**
